lib/matplotlib/axes/_base.py

```diff
@@ -2034,6 +2035,28 @@ def set_autoscaley_on(self, b):
         """
         self._autoscaleYon = b
 
+    @property
+    def use_sticky_edges(self):
+        """
+        When autoscaling, whether to obey all `Artist.sticky_edges`.
+
+        Default is ``True``.
+
+        Setting this to ``False`` ensures that the specified margins
+        will be applied, even if the plot includes an image, for
+        example, which would otherwise force a view limit to coincide
+        with its data limit.
+
+        The changing this property does not change the plot until
+        `autoscale` or `autoscale_view` is called.
+        """
+        return self._use_sticky_edges
+
+    @use_sticky_edges.setter
+    def use_sticky_edges(self, b):
+        self._use_sticky_edges = bool(b)
```

Comment:
Good point.  You are welcome to replace it with a comment to that effect.  I think I put the call in because we tend to have it in every function that potentially changes how the plot will look.